server/server.js

```diff
@@ -1,42 +1,22 @@
-import express from "express";
-import bodyParser from "body-parser";
-import path from "path";
-import ejs from "ejs";
-
 import devMode from "./util/dev-mode";
 import SyncAgent from "./sync-agent";
-
 import KueRouter from "./util/kue-router";
+import bodyParser from "body-parser";
```

Comment:
@mkawalek we tend to keep external dependencies imported before internal.

So every `import from "package-name"` should be before any `import from "./local-file"`